go/obscuronode/enclave/enclave.go

```diff
@@ -386,6 +389,33 @@ func (e *enclaveImpl) IsInitialised() bool {
 	return e.storage.FetchSecret() != nil
 }
 
+func (e *enclaveImpl) findRoundWinner(receivedRollups []*Rollup, parent *Rollup, parentState *State) (*Rollup, *State, bool) {
```

Comment:
let's leave this in the "state" file to make it easier to review. And leave a todo to move them around.

Reply:
I think this might be outdated code from before I merged the latest main 🤔  Sorry if that's the case.

Reply:
the general rule is: "Either move a function or change it in place", never "move and change".

Reply:
Yeah I don't think this has changed in this PR